packages/editor/src/utils/url.js

```diff
@@ -39,5 +39,8 @@ export function getWPAdminURL( page, query ) {
  * @return {string} Processed string
  */
 export function cleanForSlug( string ) {
+	if ( ! string ) {
+		return;
```

Comment:
I think either `@return {string}` needs to get updated in the JSDoc to signal that it can return `undefined` value or this line can change an empty string.

Reply:
Good catch! I opted for the empty string instead.